apps/main/src/llamalend/features/user-position-history/cells/EventTypeCell.tsx

```diff
@@ -0,0 +1,42 @@
+import type { ReactNode } from 'react'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import type { CellContext } from '@tanstack/react-table'
+import { CrossCircleIcon } from '@ui-kit/shared/icons/CrossCircleIcon'
+import { DownloadIcon } from '@ui-kit/shared/icons/DownloadIcon'
+import { ExclamationTriangleIcon } from '@ui-kit/shared/icons/ExclamationTriangleIcon'
+import { MinusCircleIcon } from '@ui-kit/shared/icons/MinusCircleIcon'
+import { PlusCircleIcon } from '@ui-kit/shared/icons/PlusCircleIcon'
+import { RewardsIcon } from '@ui-kit/shared/icons/RewardsIcon'
+import { UploadIcon } from '@ui-kit/shared/icons/UploadIcon'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import type { ParsedUserCollateralEvent, UserCollateralEventType } from '../hooks/useUserCollateralEvents'
+import { HistoryTableCell } from './HistoryTableCell'
+
+const { Spacing } = SizesAndSpaces
+
+const icons: Record<UserCollateralEventType, ReactNode> = {
+  'Open Position': <RewardsIcon sx={(t) => ({ color: t.design.Text.TextColors.Highlight })} />,
```

Comment:
I think we can use the string constants here instead of the `(t) =>` variants? Stuff like `textSecondary`